### DISCUSSION TITLE
why you still don't have such a function like in visual basic

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-cpptools#8103] [Ideas]
![constant](https://user-images.githubusercontent.com/79205604/132115982-56963925-849f-4477-9d26-a6b9bb0a2ff4.gif)

**Comment by sean-mcmanus** [comment]
The closest we support is 
![image](https://user-images.githubusercontent.com/19859882/132401035-3c3e63a2-5173-41ca-80fb-2ef5ef8d814b.png)

At the call site, the possible values are actually greater than just those constants, i.e. the user may use some function that evaluates to one of the constant values.

**Reply by ghost** [reply]
I tried it works, but only for C ++ in C there is no such possibility, plus you must first write "foo ::" that is, the name ENUM is also very far from the convenience of Visual basic
